livekit_internal.proto

```diff
@@ -64,6 +64,12 @@ message NodeStats {
   float load_avg_last5min = 19;
   float load_avg_last15min = 20;
   float cpu_load = 21;
+
+  // retransmissions
+  uint64 retransmit_bytes_out = 22;
+  uint64 retransmit_packets_out = 23;
+  float retransmit_bytes_out_per_sec = 24;
+  float retransmit_packets_out_per_sec = 25;
```

Comment:
should we also think about in bound retransmit stats?

Reply:
You mean the inbound retransmission stats are not actionable? We can tweak NACKs to control that a bit. But, it is not a significant amount. So, fine leaving it out for now.

Reply:
Yeah, that's what I meant.. it seems that it wouldn't have a huge impact today. We can revisit and add in the future if needed.